README.md

```diff
@@ -1 +1,30 @@
-"# todo_list" 
+# Awesome Books
+
+> This project is as implified todo list crud.
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
```

Comment:
Our Readme should follow the template which Microverse provides for us. We should not omit sections from it. :sun_with_face: 

- [ ] Please introduce the sections missing from your README, and please make sure to not miss the steps needed to replicate your app on another machine. :100: 

Microverse readme template you can find it [here](https://github.com/microverseinc/readme-template).